sdk/servicebus/service-bus/samples/v7/javascript/receiveMessagesLoop.js

```diff
@@ -5,7 +5,7 @@
  * This sample demonstrates how the receiveMessages() function can be used to receive Service Bus
  * messages in a loop.
  *
- * Setup: Please run "sendMessages.ts" sample before running this to populate the queue/topic
+ * Setup: Please run "sendMessages.js" sample before running this to populate the queue/topic
```

Comment:
Good catch, @Charlie-mwaniki.

But this is a generated file.
Can you instead update the sample under [samples-dev/receiveMessagesLoop.ts] to not have the file extension?

Updating that file and regenerating the samples in `samples/v7/javascript` and `samples/v7/typescript` would improve this.

Reply:
Hello @HarshaNalluru. I have removed the TS file extension as advised and pushed the commit. 
Thank you.

Reply:
cc @witemple-msft